packages/fuels-core/src/types/errors.rs

```diff
@@ -35,6 +35,8 @@ pub enum Error {
         revert_id: u64,
         receipts: Vec<Receipt>,
     },
+    #[error("Transaction was squeezed out")]
+    SqueezedOutTransactionError,
```

Comment:
Can we propagate the reason for why the tx was squeezed out?

Reply:
great catch!